core/templates/dev/head/i18n.js

```diff
@@ -75,8 +75,9 @@ oppia.config([
   var availableLanguageKeys = [];
   var availableLanguageKeysMap = {};
   for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
-    availableLanguageKeys.push(prop);
-    availableLanguageKeysMap[prop + '*'] = prop;
+    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
```

Comment:
Thank you for fixing this! 

And sorry I was not able to fix this myself!

After this everything should work fine :)